docs/installation.rst

```diff
@@ -221,57 +280,63 @@ Build library and tests
+``gemm/gemm_PGR0_LB0_MP0_MB_NC-*``              A modified GEMM operation where each wave targets a sub-grid of output blocks using rocWMMA API
+``gemm/gemm_PGR1_LB2_MP0_MB_CP_BLK-*``          A modified GEMM operation where each wave targets a sub-grid of output blocks using LDS memory, rocWMMA API, and block-level collaboration
+``gemm/gemm_PGR1_LB2_MP0_MB_CP_WV-*``           A modified GEMM operation where each wave targets a sub-grid of output blocks using LDS memory, rocWMMA API, and wave-level collaboration
+``gemm/gemm_PGR1_LB2_MP0_MB_CP_WG-*``           A modified GEMM operation where each wave targets a sub-grid of output blocks using LDS memory, rocWMMA API, and workgroup-level collaboration
+``gemm/gemm_PGR0_LB0_MP0_SB_NC_ad_hoc-*``       An adhoc version of ``gemm_PGR0_LB0_MP0_SB_NC-*``
+``gemm/gemm_PGR0_LB0_MP0_MB_NC_ad_hoc-*``       An adhoc version of ``gemm_PGR0_LB0_MP0_MB_NC-*``
+``gemm/gemm_PGR1_LB2_MP0_MB_CP_BLK_ad_hoc-*``   An adhoc version of ``gemm_PGR1_LB2_MP0_MB_CP_BLK-*``
+``gemm/gemm_PGR1_LB2_MP0_MB_CP_WV_ad_hoc-*``    An adhoc version of ``gemm_PGR1_LB2_MP0_MB_CP_WV-*``
+``gemm/gemm_PGR1_LB2_MP0_MB_CP_WG_ad_hoc-*``    An adhoc version of ``gemm_PGR1_LB2_MP0_MB_CP_WG-*``
+``unit/contamination_test``                     Tests against contamination of pristine data for loads and stores
+``unit/cross_lane_ops_test``                    Tests cross-lane vector operations
+``unit/fill_fragment_test``                     Tests fill_fragment API function
+``unit/io_shape_test``                          Tests input and output shape meta data
+``unit/io_traits_test``                         Tests input and output logistical meta data
+``unit/layout_test``                            Tests accuracy of internal matrix layout patterns
+``unit/load_store_matrix_sync_test``            Tests ``load_matrix_sync`` and ``store_matrix_sync`` API functions
+``unit/load_store_matrix_coop_sync_test``       Tests ``load_matrix_coop_sync`` and ``store_matrix_coop_sync`` API functions
+``unit/map_util_test``                          Tests mapping utilities used in rocWMMA implementations
+``unit/pack_util_test``                         Tests vector packing utilities used in rocWMMA implementations
+``unit/transforms_test``                        Tests transform utilities used in rocWMMA implementations
+``unit/unpack_util_test``                       Tests vector un-packing utilities used in rocWMMA implementations
+``unit/vector_iterator_test``                   Tests internal vector storage iteration implementation
+``unit/vector_test``                            Tests internal vector storage implementation
+``unit/vector_util_test``                       Tests internal vector manipulation utilities implementation
+============================================= ===================================================================================================================================================
+
```

Comment:
render result of these lines:
![image](https://github.com/ROCm/rocWMMA/assets/142121551/f1877c08-23d7-4560-80b4-e07fc19984cc)

expect a Note

![image](https://github.com/ROCm/rocWMMA/assets/142121551/033f87bf-4fed-42f0-a9ce-6e04269ce8f5)